android/src/main/java/com/rudderstack/android/internal/RudderPreferenceManager.kt

```diff
@@ -175,4 +177,18 @@ internal class RudderPreferenceManager(application: Application,
 
     val optStatus: Boolean
         get() = preferences.getBoolean(RUDDER_OPT_STATUS_KEY.key, false)
+
+    fun saveVersionName(versionName: String) {
+        preferences.edit().putString(RUDDER_APPLICATION_VERSION_KEY, versionName).apply()
+    }
+
+    val versionName: String?
+        get() = preferences.getString(RUDDER_APPLICATION_VERSION_KEY, null)
+
+    fun saveBuild(versionCode: Int) {
```

Comment:
I think should be changed to saveVersionCode

Reply:
We have decided to keep `build` instead of `versionCode`.